Cyberiad Bridge remap

S34NW opened the pull request:
This really needs a TM to ensure things are actually better this way.

<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicilty asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
Partially ports the cyberiad bridge remap from https://github.com/ss220club/Paradise-SS220/pull/199

Moves AI upload out of bridge, placing it in science and incorporates some measures to toughen defenses.
Moves gravity generator out of bridge, placing it near engineering where it is exposed to asteroids and maintenance.
Adds NSS Cyberiad statue by @dj-34
Messaging server room is now rad-proof so bridge residents can shelter safely.
(The fish in the fish pools are immune to random sentience.)
<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game
Cyberiad bridge has long had a few issues that I've previously tried to tackle (See: https://github.com/ParadiseSS13/Paradise/pull/16406)
The main point that carries over from that concept is the AI upload location:

- Currently, AI upload is completely and utterly inaccessible to antags. A large part of this is its proximity to the bridge, anyone on the bridge is currently instantly alerted to an intruder. This encourages subverting AI via a hidden upload, and leads to a ping-pong law change battle.

Other benefits include a nicer more open feel, with more entry routes into the bridge that aren't in attack range of hobos.

Gravity generator also benefits from being in a more remote location for antag play and meteor exposure.
<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes
Updated as of 2023/11/06
![2023-11-06 18 57 21](https://github.com/ParadiseSS13/Paradise/assets/12197162/e5c19fd0-dfb8-4948-a54f-40d1cbc467ba)
____
![2023-11-04 15 49 02](https://github.com/ParadiseSS13/Paradise/assets/12197162/3f863680-5507-470d-ab3f-a7f7abb423c5)
___
![2023-11-06 18 59 35](https://github.com/ParadiseSS13/Paradise/assets/12197162/78eb88a6-a3e3-4dfe-bfcf-2fc5fcf9eace)

https://github.com/ParadiseSS13/Paradise/assets/12197162/e733f77f-02b2-4c8c-aa9e-0717f19bc05c


<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Testing
Compiled and observed the map, needs playtesting with actual players.
<!-- How did you test the PR, if at all? -->

## Changelog
:cl: dj-34, S34N
tweak: Remapped Cyberiad bridge area, moved AI upload next to vault, moved explorer room to replace abandoned shop front.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

Komrad822 commented:
I like it but i feel the grav gen there will cause more harm than good since command use ai upload to escape rad storms at the moment, where would they run to with this change? The irradiated grav gen area? 

Or the grav gen gets disabled and has to be restarted, oh lovely the CE walks out greener than a christmas tree and nian/kiban captain keels over from the slightest rad exposure (very funny ngl).

Lastly, antag man steals rd id (or antag RD) but no longer needs to make their own discrete ai upload, just has to walk into a room thats out of sight and mind from command and sec. 

Like the ideas but i see flaws that need to be addressed or taken into consideration in case they havent already.

Fordoxia commented:
AI upload is currently accessible by cutting through the gravity generator's rear wall. The gravity generator only emits radiation when it's turning on or off, so it's very risk free, no one will be looking at you. It's my favourite way in to upload!

S34NW commented:
Thanks for all the comments so far, they are very useful!

To answer some points:

- HoP+Meeting room will be swapped with Captain+Teleporter
- Grav gen could be made rad-proof to give the shelter option to command
- I want to move AI upload elsewhere (near sci somewhere like cere?) and move things about again to restore the abandoned store since that is used a lot for RP shopkeeper stuff

McRamon commented:
Can you remove plants from AI upload, they completely dotn fit the place

S34NW commented:
Major updates to layout and AI upload implemented

Sadhorizon commented:
Just to make sure, explosions from the EXPERIMENTOR will not damage the upload, correct?

warriorstar-orion commented:
This looks really sick in game. Few small things:

This air alarm is probably meant to be an east bump

![2023_10_27__06_39_28__paradise dme  cyberiad dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/2eec2e23-6d0e-4e7c-95c2-312b0f001607)

This tile is missing the grass patch floor

![2023_10_27__06_39_38__paradise dme  cyberiad dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/89c16003-75a7-45b2-a042-ebf33ffb82cd)

Move the light off the glass? Not really a window so up to you

![2023_10_27__06_42_16__paradise dme  cyberiad dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/60c4df09-5008-459b-b630-373828df13b6)

S34NW commented:
> Just to make sure, explosions from the EXPERIMENTOR will not damage the upload, correct?
> 
> They are rather close and EXPERIMENTOR has a tendency to blow up sometimes, primarily with the strange object bombs.
> 
> I highly doubt anything would happen but better to be sure!

Will be good to find this out in a TM

S34NW commented:
> This looks really sick in game. Few small things:
> 
> This air alarm is probably meant to be an east bump
> 
> ![2023_10_27__06_39_28__paradise dme cyberiad dmm - StrongDMM](https://user-images.githubusercontent.com/59303604/278618357-2eec2e23-6d0e-4e7c-95c2-312b0f001607.png)
> 
> This tile is missing the grass patch floor
> 
> ![2023_10_27__06_39_38__paradise dme cyberiad dmm - StrongDMM](https://user-images.githubusercontent.com/59303604/278618485-89c16003-75a7-45b2-a042-ebf33ffb82cd.png)
> 
> Move the light off the glass? Not really a window so up to you
> 
> ![2023_10_27__06_42_16__paradise dme cyberiad dmm - StrongDMM](https://user-images.githubusercontent.com/59303604/278618573-60c4df09-5008-459b-b630-373828df13b6.png)

Many thanks, addressed

Denthamos commented:
For the gravity generator it is kinda inefficient and not fitting ICly to have the radsuits inside the room which gives off the radiation instead of having a prep room like it does currently.  Is there anyway to keep a suiting up room for it with the suits in it?  That room could also be made into the radiation proof room from rad storms as a scrubbing room instead of making the entire grav gen rad proof.

Otherwise I do like how this looks.  Very fancy.

Henri215 reviewed (changes requested):
1. Captain quarters is missing the captain rubber ducky
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/08d05064-d822-4ab2-b083-d86f01c4d401)

2. One can access that extinguisher cabinet?
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/adb4ee72-7371-47b8-b5fc-923daacef647)

3. This locker shouldnt be using the EAST version?
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/42e3639c-65f4-4e3b-9f3f-88dbfe8c4fd3)

4. This pipe placed by accident here

![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/1f59f680-1256-4a54-ad68-dfe4ffabf929)

5. No APC inside this area
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/edeab767-7813-4c53-81af-743d2e562a2e)

6. Two cameras right next to each other, not necessary
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/046052e2-2bc6-43e0-8df4-f8950c17a0e1)

7. Could give an organised placement to the AI modules on the upload chamber table

8. Same here, you can access that air alarm on the meeting room?
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/66a52e5e-1b6f-45f5-b6f2-216ab85c3436)

9. Request console placed by accident, only one is necessary and its blocking the keycard authentication device
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/aba75414-ca10-452d-8520-ef1762a87714)

S34NW commented:
> 1. Captain quarters is missing the captain rubber ducky
>        ![image](https://user-images.githubusercontent.com/77684085/280154014-08d05064-d822-4ab2-b083-d86f01c4d401.png)
> 
>     2. One can access that extinguisher cabinet?
>        ![image](https://user-images.githubusercontent.com/77684085/280154495-adb4ee72-7371-47b8-b5fc-923daacef647.png)
> 
>     3. This locker shouldnt be using the EAST version?
>        ![image](https://user-images.githubusercontent.com/77684085/280155123-42e3639c-65f4-4e3b-9f3f-88dbfe8c4fd3.png)
> 
>     4. This pipe placed by accident here
> 
> 
> ![image](https://user-images.githubusercontent.com/77684085/280156580-1f59f680-1256-4a54-ad68-dfe4ffabf929.png)
> 
>     5. No APC inside this area
>        ![image](https://user-images.githubusercontent.com/77684085/280156060-edeab767-7813-4c53-81af-743d2e562a2e.png)
> 
>     6. Two cameras right next to each other, not necessary
>        ![image](https://user-images.githubusercontent.com/77684085/280156747-046052e2-2bc6-43e0-8df4-f8950c17a0e1.png)
> 
>     7. Could give an organised placement to the AI modules on the upload chamber table
> 
>     8. Same here, you can access that air alarm on the meeting room?
>        ![image](https://user-images.githubusercontent.com/77684085/280157153-66a52e5e-1b6f-45f5-b6f2-216ab85c3436.png)
> 
>     9. Request console placed by accident, only one is necessary and its blocking the keycard authentication device
>        ![image](https://user-images.githubusercontent.com/77684085/280157528-aba75414-ca10-452d-8520-ef1762a87714.png)

Fixed all points bar 3. Point 3 is an issue with the east/west sprites for the cabinet being flipped.

Bjamcham commented:
<details>
<summary>A few bits I'm curious about</summary>
<br>
1. Could the cap air alarm and fire alarm possibly shift over to the north and west wall just to clear it up?

2. The maints room seems like a really vulnerable bridge access point with both the bridge and bedroom walls as normal walls. Breaking into the cap's office is now only two subtle welds away.

![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/cef0c587-eeb0-4b6b-8a0a-d1fb2006d679)

3. With the new fire extinguisher sprites, would it be better to shift the cabinet one wall left in the meeting room?
![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/2e4c5d52-e8c3-4c02-a26e-4ff19908a74e)

4. Would these lights be better on the walls just north and south?
![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/2de67faf-5eb8-467b-a2f9-a3c50a157a43)

5. Was the meeting room intended not to have any electrified grilles? 
6. `Gravity Generator prep room`, should prep and room be capitalized?
7.  This may just be me but this section looks really crowded with wall mounts. Maybe move the grav gen fire alarm to an east/west wall, the bridge apc and air alarms could fit in on the actual wrap around hallway, exposing a potential bit of sabotage against the final bridge itself.
![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/fe6dd184-ca70-4342-b27b-b5498bfd0da6)

8. Would the cam be better off on a wall above or below?
![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/836566b0-d2bc-4038-8eb5-c3fc67691c0e)

9. This section of walls is very spartan although I'm not sure what could be added.
![image](https://github.com/ParadiseSS13/Paradise/assets/107899953/791725f5-f5fe-4e73-82bf-1c71490b02af)

</details>

Banderblitz commented:
I'll add my own two cents on this since I frequently command main and or deal with AI shenanigans.

Bridge area is nice except for two key points.

1. Engineering should still have access to the gravity generator to fix it without requiring a CE as most often there *isnt* one. Flip it around and give engineers access to the bottom two lower bridge doors and I think that is fixed.
2. Captain's bedroom has a single normal wall to break into right next his bed and the locker, I get that things being too secure is problematic, but a *single* normal wall to get the captain's locker is too easy... every single head of staff's locker is harder to get to due to being public or at LEAST a reinforced one and that should be fixed.

Now AI upload: I get the whole pingponging AI laws because of a convenient to access upload for captain to use, but making it FAR too inconvenient to use and out of the way isnt how to fix that. People that want to subvert an AI or command that want an AI upload  will still do so by getting into tech storage, it being so far out of the way in science just means less effort to try and subvert it, and significantly harder to actually fix it because it's just out of the way. The AI upload should at LEAST be as close as the RD office is to the main primary hallway now, if not closer IMO for 'security' sake since it's the AI upload it shouldn't be a remote room. Even if it requires shifting the entirety of science on the same side down. I believe it's easily doable to fit it closer while not making it so far out of the way that subverting it is as easy as breaking down one wall where noone ever is, and changing laws.

ItsMarmite commented:
scientists can turn off the AI upload turret control
RD remote can't interact with the doors

S34NW commented:
> A few bits I'm curious about
> 
> 1. Could the cap air alarm and fire alarm possibly shift over to the north and west wall just to clear it up?
> 
>     2. The maints room seems like a really vulnerable bridge access point with both the bridge and bedroom walls as normal walls. Breaking into the cap's office is now only two subtle welds away.
> 
> 
> ![image](https://user-images.githubusercontent.com/107899953/280420187-cef0c587-eeb0-4b6b-8a0a-d1fb2006d679.png)
> 
>     3. With the new fire extinguisher sprites, would it be better to shift the cabinet one wall left in the meeting room?
>        ![image](https://user-images.githubusercontent.com/107899953/280419963-2e4c5d52-e8c3-4c02-a26e-4ff19908a74e.png)
> 
>     4. Would these lights be better on the walls just north and south?
>        ![image](https://user-images.githubusercontent.com/107899953/280420424-2de67faf-5eb8-467b-a2f9-a3c50a157a43.png)
> 
>     5. Was the meeting room intended not to have any electrified grilles?
> 
>     6. `Gravity Generator prep room`, should prep and room be capitalized?
> 
>     7. This may just be me but this section looks really crowded with wall mounts. Maybe move the grav gen fire alarm to an east/west wall, the bridge apc and air alarms could fit in on the actual wrap around hallway, exposing a potential bit of sabotage against the final bridge itself.
>        ![image](https://user-images.githubusercontent.com/107899953/280422483-fe6dd184-ca70-4342-b27b-b5498bfd0da6.png)
> 
>     8. Would the cam be better off on a wall above or below?
>        ![image](https://user-images.githubusercontent.com/107899953/280422754-836566b0-d2bc-4038-8eb5-c3fc67691c0e.png)
> 
>     9. This section of walls is very spartan although I'm not sure what could be added.
>        ![image](https://user-images.githubusercontent.com/107899953/280424057-791725f5-f5fe-4e73-82bf-1c71490b02af.png)

1. Fixed
2. Changed access to atmos only, so hackers have to go through the door or weld rwall, can revisit this if it's still a problem
3. Fixed
4. Fixed
5. Yes as it has shutters
6. Good shout, fixed
7. Moved as suggested, fixed
8. Fixed
9. Fixed somewhat

S34NW commented on code/game/area/ss13_areas/engineering_areas.dm:
```diff
@@ -50,6 +50,9 @@
 	icon_state = "gravgen"
 
+/area/station/engineering/gravitygenerator/entry
+	name = "\improper Gravity Generator prep room"
```
```suggestion
	name = "\improper Gravity Generator Prep Room"
```

S34NW commented:
> I'll add my own two cents on this since I frequently command main and or deal with AI shenanigans.
> 
> Bridge area is nice except for two key points.
> 
>     1. Engineering should still have access to the gravity generator to fix it without requiring a CE as most often there _isnt_ one. Flip it around and give engineers access to the bottom two lower bridge doors and I think that is fixed.
> 
>     2. Captain's bedroom has a single normal wall to break into right next his bed and the locker, I get that things being too secure is problematic, but a _single_ normal wall to get the captain's locker is too easy... every single head of staff's locker is harder to get to due to being public or at LEAST a reinforced one and that should be fixed.
> 
> 
> Now AI upload: I get the whole pingponging AI laws because of a convenient to access upload for captain to use, but making it FAR too inconvenient to use and out of the way isnt how to fix that. People that want to subvert an AI or command that want an AI upload will still do so by getting into tech storage, it being so far out of the way in science just means less effort to try and subvert it, and significantly harder to actually fix it because it's just out of the way. The AI upload should at LEAST be as close as the RD office is to the main primary hallway now, if not closer IMO for 'security' sake since it's the AI upload it shouldn't be a remote room. Even if it requires shifting the entirety of science on the same side down. I believe it's easily doable to fit it closer while not making it so far out of the way that subverting it is as easy as breaking down one wall where noone ever is, and changing laws.

1. Is a valid point, I'm going to have to think more about how we can make this work better, thank you.
2. I've mitigated this with removing maint access and applying atmos access to the door, if this continues to be an issue I will revisit the wall setup
3. AI upload is going to be fine here, on Cere it's even further from RD and much more remote, but is still accessible and has worked perfectly fine there. This implementation trades some of the security of the cere upload (double rwall) for proximity to people. There is almost always someone in science at any given point so it gets security on that front.

S34NW commented:
> scientists can turn off the AI upload turret control
> RD remote can't interact with the doors

1. This was an issue from copy pasting the turret controller, likely exists as a bug on current cyberiad just not noticed by anyone. Fixed.
2. This is the same as cyberiad, only command remote has access to the door. Tweaks to the remote needed for that.

Freakycrafter commented:
![grafik](https://github.com/ParadiseSS13/Paradise/assets/57047909/0c4ec55e-7230-4989-a408-6fc324dccbc7)
The bridge's power monitor isnt connected yet.

S34NW commented:
> ![grafik](https://user-images.githubusercontent.com/57047909/280438886-0c4ec55e-7230-4989-a408-6fc324dccbc7.png) The bridge's power monitor isnt connected yet.

Fixed

S34NW commented:
Major changes to the remap implemented, see images.

go-liath-online commented:
Loving this remap so far. A few nitpicks:

- AI upload has two holopads, which is probably unnecessary for such a small room. Even atmos only has one!
- AI upload is missing AI status displays. Might flesh the room out a bit better.
- AI upload intercom is on General, whereas in the old upload, it was on Private.
- Gravity generator holopad is in a strange spot.
- The new placement of the camera aft of the bridge means AI has a blind spot between the Cyberiad sculpture and the aft hallway.
- The intercom in the meeting room isn't in the middle of the table anymore.
- The red phone in the heads of staff meeting room is in Russian.

S34NW commented:
> Loving this remap so far. A few nitpicks:
> 
>     * AI upload has two holopads, which is probably unnecessary for such a small room. Even atmos only has one!
> 
>     * AI upload is missing AI status displays. Might flesh the room out a bit better.
> 
>     * AI upload intercom is on General, whereas in the old upload, it was on Private.
> 
>     * Gravity generator holopad is in a strange spot.
> 
>     * The new placement of the camera aft of the bridge means AI has a blind spot between the Cyberiad sculpture and the aft hallway.
> 
>     * The intercom in the meeting room isn't in the middle of the table anymore.
> 
>     * The red phone in the heads of staff meeting room is in Russian.

Thanks, a fix is in progress for all of these points.

matttheficus commented:
Personal nitpicks:

- This double-wall looks off to me. Not a fan of double-walls to begin with.
![image](https://github.com/ParadiseSS13/Paradise/assets/57759731/77581c46-eef1-47ee-9537-dc2abb02c1a7)

- This entire hallways is now a nightmare of MetaStation proportions - it is for all intents a 1-tile wide hallway and make a significant chokepoint in SciMaints compared to the usual version with the "spare room". Getting cornered here is a death sentence.
![image](https://github.com/ParadiseSS13/Paradise/assets/57759731/8ad09f85-a6ac-4b63-bf66-032c58da3b73)

- Some of the Bridge doors are missing firelocks.

matttheficus commented:
![image](https://github.com/ParadiseSS13/Paradise/assets/57759731/4b94c942-e36f-4335-b92f-c7cde2cb7d96)

The turrets in the Upload are now half useless. They cannot hit anyone walking into the doors, and if set to lethal, will just break things.

matttheficus commented:
The Bridge APC now runs out of power VERY VERY VERY quickly before the Engine/SMESs are set up - even faster than Cargo's APC.

S34NW commented:
> Personal nitpicks:
> 
>     * This double-wall looks off to me. Not a fan of double-walls to begin with.
>       ![image](https://user-images.githubusercontent.com/57759731/280575782-77581c46-eef1-47ee-9537-dc2abb02c1a7.png)
> 
>     * This entire hallways is now a nightmare of MetaStation proportions - it is for all intents a 1-tile wide hallway and make a significant chokepoint in SciMaints compared to the usual version with the "spare room". Getting cornered here is a death sentence.
>       ![image](https://user-images.githubusercontent.com/57759731/280576318-8ad09f85-a6ac-4b63-bf66-032c58da3b73.png)
> 
>     * Some of the Bridge doors are missing firelocks.

Addressed



> The Bridge APC now runs out of power VERY VERY VERY quickly before the Engine/SMESs are set up - even faster than Cargo's APC.

Addressed



> ![image](https://user-images.githubusercontent.com/57759731/280578376-4b94c942-e36f-4335-b92f-c7cde2cb7d96.png)
> 
> The turrets in the Upload are now half useless. They cannot hit anyone walking into the doors, and if set to lethal, will just break things.

Bit of a tricky one, but attempted to address.

go-liath-online commented:
Heads up, you have `/obj/machinery/status_display/supply_display` in the AI upload instead of `/obj/machinery/ai_status_display` so the screen in there just shows the cargo shuttle timer.

Git-GoR reviewed (changes requested):
<details><summary>Some issues in Bridge area:</summary>

1.  Teleporter room missing the yellow caution stripes.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/e75e136d-3e2f-4f7d-8a61-2f12c47f68b1)

1. In the eastern SE Central Hallway you've added a fire alarm, when there was already one a few tiles north, right before the corner.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/5c9434a9-3046-41f9-82b1-362902cf4686)

1. HoP's Office Door button needs to be linked to the airlock.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/497f86a2-0bdb-4e8b-8966-bf95e5db9f28)

1. The Privacy Shutters on the table of HoP's office have a `dir` 8, making them close the other way around of the rest in the same line.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/0bdd4792-8f4f-452c-adef-bd3e1310d120)

1. Even with night lighting off, the south entrance of the Bridge Lounge is kinda dark. It could use a lightbulb. This would also help greatly with the visibility in the lower Lounge during night lights as a whole, as currently is sort of dark.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/f540531a-f97f-4e9c-9a6f-d8d07e4aa7e7)

1. Couple of camera darkspots in the Lounge south entrance. Not the biggest deal, but could be a small nuisance as AI. (I wonder if non room wide darkspots would be a mechanic worth adding to other areas)
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/01166e47-d68c-41c5-8e21-4274bb0ff1fe)

1. Lounge has a very low intercom coverage.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/3828ad6d-38df-4cda-a5cd-5d7142d375b5)

1. Meeting room has no fire alarm.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/6cbda6ed-f06f-4ece-bb38-b0eec2269b68)

1. Lounge has no fire extinguisher cabinets (Greygang buff :alien:)
</details>

<details><summary>Single thing in Gravgen:</summary>

1. The Gravity Generator SMES is now connected to the general powernet throught the input, intended? While this will not impact it with a powersink, it might add a bit of power requirement to the station as a whole as it's another SMES that needs to be charged.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/62d954ee-2ceb-4b2b-8d55-208a851873c6)

</details>

<details><summary>Couple things in RnD Area:</summary>

1. Experimentor room lacks the button to close the protective shutters.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/b483b51e-1f3d-4044-a1dc-ceb1992003da)

1. The Camera console and the camera itself are added to the network... `Telepad`? The previous one was in the `Experimientator` net, judging by the prefabs. It works, but makes little sense.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/1b51787c-a2f0-4157-b595-a38e4fc0adb1)

</details>

S34NW commented:
> Some issues in Bridge area:
> 
>     1. Teleporter room missing the yellow caution stripes.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281689690-e75e136d-3e2f-4f7d-8a61-2f12c47f68b1.png)
> 
>     2. In the eastern SE Central Hallway you've added a fire alarm, when there was already one a few tiles north, right before the corner.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281693950-5c9434a9-3046-41f9-82b1-362902cf4686.png)
> 
>     3. HoP's Office Door button needs to be linked to the airlock.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281697023-497f86a2-0bdb-4e8b-8966-bf95e5db9f28.png)
> 
>     4. The Privacy Shutters on the table of HoP's office have a `dir` 8, making them close the other way around of the rest in the same line.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281697687-0bdd4792-8f4f-452c-adef-bd3e1310d120.png)
> 
>     5. Even with night lighting off, the south entrance of the Bridge Lounge is kinda dark. It could use a lightbulb. This would also help greatly with the visibility in the lower Lounge during night lights as a whole, as currently is sort of dark.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281701794-f540531a-f97f-4e9c-9a6f-d8d07e4aa7e7.png)
> 
>     6. Couple of camera darkspots in the Lounge south entrance. Not the biggest deal, but could be a small nuisance as AI. (I wonder if non room wide darkspots would be a mechanic worth adding to other areas)
>        ![imagen](https://user-images.githubusercontent.com/58085266/281709301-01166e47-d68c-41c5-8e21-4274bb0ff1fe.png)
> 
>     7. Lounge has a very low intercom coverage.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281710182-3828ad6d-38df-4cda-a5cd-5d7142d375b5.png)
> 
>     8. Meeting room has no fire alarm.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281711662-6cbda6ed-f06f-4ece-bb38-b0eec2269b68.png)
> 
>     9. Lounge has no fire extinguisher cabinets (Greygang buff 👽)
> 
> 
> Single thing in Gravgen:
> 
>     1. The Gravity Generator SMES is now connected to the general powernet throught the input, intended? While this will not impact it with a powersink, it might add a bit of power requirement to the station as a whole as it's another SMES that needs to be charged.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281715023-62d954ee-2ceb-4b2b-8d55-208a851873c6.png)
> 
> 
> Couple things in RnD Area:
> 
>     1. Experimentor room lacks the button to close the protective shutters.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281716965-b483b51e-1f3d-4044-a1dc-ceb1992003da.png)
> 
>     2. The Camera console and the camera itself are added to the network... `Telepad`? The previous one was in the `Experimientator` net, judging by the prefabs. It works, but makes little sense.
>        ![imagen](https://user-images.githubusercontent.com/58085266/281717831-1b51787c-a2f0-4157-b595-a38e4fc0adb1.png)

All points fixed. Although as a note, the cam console and experimentor cam net were unchanged by me. Not sure what happened there 😕

Git-GoR reviewed (changes requested):
All good, just one small thing. Due to the junction to HoP's office, the Meeting Room disposals will throw all the trash back out inmediately.
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/04de92e6-74d3-4063-bf74-fc04c317518d)

Could be solved by doing something like this, or whatever you come up with (disposals pain):
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/ba75b965-081d-402b-8a46-e020e007f98a)

Once that is fixed, it should be good to go.

S34NW commented:
> All good, just one small thing. Due to the junction to HoP's office, the Meeting Room disposals will throw all the trash back out inmediately.
> ![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/04de92e6-74d3-4063-bf74-fc04c317518d)
> 
> Could be solved by doing something like this, or whatever you come up with (disposals pain):
> ![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/ba75b965-081d-402b-8a46-e020e007f98a)
> 
> Once that is fixed, it should be good to go.
> 

Have you confirmed this? It was my understanding that trash hitting specifically sort junctions from the reverse direction will pass as expected. This is the behaviour in a number of places on-station as per my understanding.

Git-GoR commented:
I was not sure myself as well, so I tested it by chucking a random item from the room into the chute, which bounced out one second later. I tried disposaling myself in, same result.

I did not test a tagged item, maybe in that case works different? In any case, please double check just to fully confirm it.

S34NW commented:
> I was not sure myself as well, so I tested it by chucking a random item from the room into the chute, which bounced out one second later. I tried disposaling myself in, same result.
> 
> I did not test a tagged item, maybe in that case works different? In any case, please double check just to fully confirm it.

Fixed

Contrabang commented:
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/853a90e4-e98e-4dfc-b4b7-eb5b9c2d690d)
This APC is disconnected

Henri215 commented:
Its looking good to me, just need to fix that APC issue

S34NW commented:
> ![image](https://user-images.githubusercontent.com/91113370/282296476-853a90e4-e98e-4dfc-b4b7-eb5b9c2d690d.png) This APC is disconnected

fixed

lewcc reviewed (approved):
LGTM (let's get this mapped)
